plugins/system/menufilecheck/menufilecheck.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension type="plugin" group="system" method="upgrade">
+    <name>System - Menu File Check</name>
+    <author>BK Design Solutions</author>
+    <creationDate>June 2024</creationDate>
+    <copyright>Copyright (C) 2024 BK Design Solutions</copyright>
+    <license>Private</license>
+    <authorEmail>bh@bkdesignsolutions.com</authorEmail>
+    <authorUrl>https://bkdesignsolutions.com</authorUrl>
+    <version>1.0.0</version>
```

Comment:
Since our component needs this plugin to work it will be part of the bundle so it should have the same version number